src/app2.js

```diff
@@ -0,0 +1,117 @@
+import axios from 'axios';
+
+console.log('Hallo daar!');
+
+
+// Schrijf een asynchrone functie die, met behulp van Axios, een GET-request maakt naar het juiste endpoint. Log de response in de console en bestudeer de data goed: hoe is het opgebouwd?
+
+// koppel index list
+const list = document.getElementById("country-list")
+
+
+async function fetchCountrySearch(countryName) {
+
+    const response = await axios.get(`https://restcountries.com/v2/name/${countryName}`);
+
+    try {
+
+        // maak de list leeg
+        list.replaceChildren()
+
+        //map door data
+        response.data.map((country) => {
+            const {flags: {png}, name, capital, subregion, currencies, population, languages} = country;
+
+
+            //flags
+            const flagCountry = document.createElement('img');
+            flagCountry.setAttribute('src', country.flag);
+            flagCountry.setAttribute('class', "country-flag")
+            list.appendChild(flagCountry)
+
+
+            //maak list element
+            const firstCountry = document.createElement('p');
+            firstCountry.textContent = country.name;
+            list.appendChild(firstCountry);
+
+            //region
+            const countryRegion = document.createElement('p');
+            countryRegion.textContent = country.region;
+            list.appendChild(countryRegion);
+
+            //population
+            const popularCountry = document.createElement('p');
+            popularCountry.textContent = `Has a population of ${country.population} people`;
+            list.appendChild(popularCountry);
+
+            //make array list of needed names.
+            let coinString;
+            const coinArray = currencies.map((coin) => {
+                return coin.name;
+            })
+            // print strings currencies
+            if (currencies.length === 1) {
+                coinString = `you can pay with ${coinArray[0]}`;
+            } else {
+                coinString = `you can pay with ${coinArray[0]} and ${coinArray[1]}`;
+            }
```

Comment:
Dit is niet echt uitbreidbaar voor als er meer dan twee talen zouden zijn. Wellicht dat je de volgende keer iets met een for-loop of de [Array.join](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join) functie kan doen